core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -195,10 +196,22 @@ private fun dateFormatted(publishDate: Instant): String {
 }
 
 @Composable
-fun NewsResourceDate(
-    publishDate: Instant
+fun NewsResourceDateAndType(
+    publishDate: Instant,
+    resourceType: NewsResourceType
+
 ) {
-    Text(dateFormatted(publishDate), style = MaterialTheme.typography.labelSmall)
+    val date = dateFormatted(publishDate)
+
+    if (resourceType != NewsResourceType.Unknown) {
+        Text(
+            "$date • ${resourceType.displayText}", style = MaterialTheme.typography.labelSmall
```

Comment:
Should this string template be in a `strings.xml` for translation purposes? I'm just thinking if a RTL language would want the order of `date` and the resource type to be swapped.